script.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+//1.Given the string 'ahb acb aeb aeeb adcb axeb'. Write a regular expression that matches the strings ahb, acb, aeb by pattern: letter 'a', any character, letter 'b'
+let str ='ahb acb aeb aeeb adcb axeb'.match(/a[a-z]b/g);
+console.log(str);
+
+//2.Given the string '2 + 3 223 2223'. Write a regexp that finds line 2 + 3 without capturing the rest
+let numStr = '2 + 3 223 2223';
+const reg =  /2...3/;
```

Comment:
it seems that "2  3", "2 - 3" and etc are valid?

Reply:
I have changed it, is now everything right?